src/CommunityToolkit.Maui.UnitTests/Behaviors/StatusBarBehaviorTests.cs

```diff
@@ -54,4 +54,24 @@ public void VerifyStatusBarStyleCallsOnPropertyChangedThrowsExceptionOnUnsupport
 		var exception = Assert.Throws<NotSupportedException>(() => statusBarBehavior.StatusBarStyle = StatusBarStyle.DarkContent);
 		exception.Message.Should().Be("PlatformSetStyle is only supported on iOS and Android 23 and later");
 	}
+
+	[Fact]
+	public void VerifyStatusBarColorAndStyleCallsOnPageNavigatedToThrowsExceptionOnUnsupportedPlatform()
+	{
+		var statusBarBehavior = new StatusBarBehavior();
+		
+		var contentPage = new Page
+		{
+			Behaviors = { statusBarBehavior }
+		};
+
+		var onAttachedTo = statusBarBehavior.GetType().GetMethod("OnAttachedTo", System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Instance | System.Reflection.BindingFlags.DeclaredOnly);
```

Comment:
I had my hopes up that OnAttachedTo would be called as it was added to the Page's list of Behaviors, as there was a unit test earlier with a name that implied that would happen. 

If there is a way to trigger the OnAttachedTo method without using reflection then please let me know, otherwise I can remove it completely.

Reply:
I removed the unit test as I could not get the `PlatformBehavior.OnAttachedTo` to get called without copy & pasting a lot of test code from dotnet/maui.